src/css/controls/imports/display.less

```diff
@@ -32,8 +34,6 @@ display icons
 }
 
 .jw-display-container {
-    display: table-cell;
```

Comment:
with this removed do we need vertical align middle?

Reply:
might need to switch to align-items: center to keep the expected behavior, make sure to check that as well.

Reply:
Looks like vertical-align was no longer needed. Nice catch!




> might need to switch to align-items: center to keep the expected behavior, make sure to check that as well.
Looks like the text-align is keeping the controls centered, so align-items doesn't seem to be needed.

Reply:
Just to clarify, align-items/vertical-align are vertical alignment properties, text-align is horizontal. CSS naming is weird.

Reply:
Right. The vertical alignment seems to be handled by the flex-direction and justify-content of the parent jw-display div.